src/backbone.virtual-collection.js

```diff
@@ -117,6 +117,7 @@ var VirtualCollection = Backbone.VirtualCollection = Backbone.Collection.extend(
         this.trigger('change', model, this, options);
       } else {
         this._indexAdd(model);
+        this.listenTo(model, 'all', this._onAllEvent);
```

Comment:
not at the moment, it's just to maintain the standard function signature of backbone.